download_deps

```diff
@@ -11,5 +11,22 @@ fi
 
 if [ ! -f libs/square-otto-1.3.2.jar ];
 then
-    curl -o libs/square-otto-1.3.2.jar https://cloud.github.com/downloads/square/otto/square-otto-1.3.2.jar
+    if [ ! -d ~/.soomla ];
+    then
+        mkdir ~/.soomla
+    fi
+    if [ ! -d ~/.soomla/cache ];
+    then
+        mkdir ~/.soomla/cache
+    fi
+    if [ ! -d ~/.soomla/cache/android-core ];
+    then
+        mkdir ~/.soomla/cache/android-core
+    fi
+
+    if [ ! -f ~/.soomla/cache/android-core/square-otto-1.3.2.jar ];
+    then
+        curl -o ~/.soomla/cache/android-core/square-otto-1.3.2.jar https://cloud.github.com/downloads/square/otto/square-otto-1.3.2.jar
```

Comment:
@EugeneButusov when downloading, download first to a temp file, maybe <filename>.tmp and then remove .tmp.
This is for cases when download is aborted by user but file is there, and it's corrupted.